app/io/pathfinder/websockets/WebSocketMessage.scala

```diff
@@ -77,16 +77,30 @@ object WebSocketMessage {
    */
   case class Delete(
     model: ModelType,
-    id:     Long
+    id:    Long
   ) extends ControllerMessage
   implicit val deleteFormat = Json.format[Delete]
 
+
+  case class Route(
+    model: ModelType,
+    id:    Long
+  ) extends ControllerMessage
+  implicit val routeFormat = Json.format[Route]
+
+  case class Routed(
```

Comment:
What is a Routed and how is it different from a Route? Please add comments explaining.